xml/containers-support.xml

```diff
@@ -91,12 +107,29 @@
     </para>
    </listitem>
   </orderedlist>
+
+  <para>
+   For further information, refer to the
```

Comment:
The stylewich committee agreed on "For more information" (even though I was in favor of "further information").